snmp/assets/dashboards/troubleshooting_dashboard.json

```diff
@@ -0,0 +1,2533 @@
+{
+    "author_name": "Datadog",
+    "description": null,
+    "layout_type": "ordered",
+    "template_variables": [
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "Host",
+            "prefix": "host"
+        }
+    ],
+    "title": "NDM Troubleshooting Dashboard",
+    "widgets": [
+        {
+            "definition": {
+                "background_color": "white",
+                "layout_type": "ordered",
+                "show_title": true,
+                "title": "NDM Troubleshooting Dashboard",
+                "type": "group",
+                "widgets": [
+                    {
+                        "definition": {
+                            "background_color": "transparent",
+                            "content": "This is a high level overview of the health of your network devices monitoring. You can track health and performance of your running agents, as well as SNMP, Traps and NetFlow integrations.\n\n**Further reading on NDM**\n\n- Our blog post on [Monitoring Datacenters and network devices with Datadog](https://www.datadoghq.com/blog/datacenter-monitoring-dashboards/)\n- Our blog post on [Monitoring network devices with SNMP](https://www.datadoghq.com/blog/monitor-snmp-with-datadog/)\n- Network Device Monitoring [docs](https://docs.datadoghq.com/network_performance_monitoring/devices) ",
+                            "font_size": "14",
+                            "has_padding": true,
+                            "show_tick": false,
+                            "text_align": "left",
+                            "tick_edge": "left",
+                            "tick_pos": "50%",
+                            "type": "note",
+                            "vertical_align": "top"
+                        },
+                        "id": 3628964508547190,
+                        "layout": {
+                            "height": 4,
+                            "width": 6,
+                            "x": 0,
+                            "y": 0
+                        }
+                    }
+                ]
+            },
+            "id": 2663583616845740,
+            "layout": {
+                "height": 5,
+                "width": 6,
+                "x": 0,
+                "y": 0
+            }
+        },
+        {
+            "definition": {
+                "background_color": "pink",
+                "layout_type": "ordered",
+                "show_title": true,
+                "title": "Overview",
```

Comment:
I'm not sure if the total count of:
- Submitted Metrics
- Traps indexed
- Flows indexed
are very interesting alone.

Those seems to be easier to read as timeseries graph (already exist elsewhere).

What about having "Subnets" instead? That said, it seems not possible to have a Query Value with count non-zero. It's possible with timeseries though: 
![image](https://github.com/DataDog/integrations-core/assets/49917914/5961064e-4781-4772-b8bf-2ed6368c3119)

Reply:
Yes, this "overview" section is mostly here to follow the good practises listed [here](https://datadoghq.dev/integrations-core/guidelines/dashboards/#general) and I had trouble finding relevant metrics for it
Subnets count is displayed at the end of the SNMP section (using a `count_not_null`), but it does make sense to be displayed in the Overview, will move it!